docs/develop/go/hello_world.md

```diff
@@ -115,7 +115,7 @@ $ wasmedge hello.wasm
 Hello TinyGo from WasmEdge!
 ```
 
-For the `--reactor` mode,
+For the `--reactor` mode:
```

Comment:
@ahndmal Comma punctuation is well suited in this case. So could you please revert that one? The other change regarding extra backtick looks good to me

Reply:
Hello @adithyaakrishna !
Sure, changed it back to comma :) Thanks